lib/brewdler/brew_installer.rb

```diff
@@ -2,14 +2,14 @@ module Brewdler
   class BrewInstaller
     def self.install(name, options = {})
       if `which brew`; $?.success?
-        command = "brew install #{name}"
+        command = [name]
         unless options[:args].nil?
           options[:args].each do |arg|
-            command << " --#{arg}"
+            command << "--#{arg}"
           end
         end
 
-        `#{command}`
+        system "brew", "install", *command
```

Comment:
You probably need to use `IO.popen` for this case. It's a bit of a nastier API but it's probably worth it longer term for the flexibility it provides.